Library/Homebrew/rubocops/version.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rubocops/extend/formula"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class Version < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          version_node = find_node_method_by_name(body_node, :version)
+          return unless version_node
+
+          version = string_content(parameters(version_node).first)
+
+          problem "version is set to an empty string" if version.blank?
```

Comment:
Can't use `.blank?` here (or anything from ActiveSupport) I'm afraid. 
```suggestion
          problem "version is set to an empty string" if version.empty?
```

Reply:
Thanks for pointing it out. Fixed with d37dbc94d5666ec39589bd65bc31cb3863306eab